packages/locales/en/common.json

```diff
@@ -1,6 +1,6 @@
 {
   "back": "",
-  "cancel": "",
+  "cancel": "Cancel",
```

Comment:
This file should always be left blank. I need to add some rule or gitignore check for this, still have to decide it.

The common strings come from the ui-components repo. This file gets created by a limitation of the script that extracts the i18n keys from the code, which doesn't understand that the common namespace is an external dependency.